site/tutorials/tutorial-two-dotnet.md

```diff
@@ -213,9 +213,9 @@ message wasn't processed fully and will redeliver it to another
 consumer. That way you can be sure that no message is lost, even if
 the workers occasionally die.
 
-There aren't any message timeouts; RabbitMQ will redeliver the message
-only when the worker connection dies. It's fine even if processing a
-message takes a very, very long time.
+There aren't any message timeouts; RabbitMQ will redeliver the message when
+the consumer dies. It's fine even if processing a message takes a very, very
```

Comment:
Ok, I'm not familiar with the technical details, I just saw #88 and the [accepted answer on Stack Overflow](http://stackoverflow.com/a/32909897/61818) which has the wording I've pull requested. Should #88 just be closed as `wontfix`, then?